src/viewing.jl

```diff
@@ -108,3 +135,13 @@ necessarily include the right value. This behavior is different than the more
 intuitive behavior of `view(object, Box((0.5,0.5), (10.0,10.0))`.
 """
 slice(object, ranges...) = view(object, Box(first.(ranges), last.(ranges)))
+
+function _fillmask!(mask, n, v)
+  for col in eachcol(mask)
+    find = findfirst(==(n), col)
+    lind = findlast(==(n), col)
+    if !isnothing(find) && !isnothing(lind)
+      col[find:lind] .= v
+    end
```

Comment:
```suggestion
    i = findfirst(==(n), col)
    j = findlast(==(n), col)
    col[i:j] .= v
```
Here we are guaranteed that there will always be a first and a last index, no need to check for nothing.